doc/hardware/porting/board_porting.rst

```diff
@@ -65,112 +65,128 @@ complete conversion reference.
 .. _example-application conversion Pull Request: https://github.com/zephyrproject-rtos/example-application/pull/58
 .. _conversion script: https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/utils/board_v1_to_v2.py
 
-.. _board_and_identifiers:
+.. _hw_support_hierarchy:
 
-Board and board identifiers
-***************************
+Hardware support hierarchy
+**************************
 
-A board may be a physical piece of hardware or an emulated board.
-Furthermore a board may contain one or multiple SoCs. Also, each SoC may contain
-one or multiple CPU clusters. A CPU cluster refers to a group of CPU cores.
-Only CPU cores of same architecture can be in the same cluster. In the case
-where a physical SoC considers a CPU cluster to contain CPU cores of different
-architectures then those must be modelled as multiple clusters, where all CPU
-cores within a cluster is having the same architecture.
-It is possible to have only a single CPU core within a CPU cluster.
+Zephyr's hardware support hierarchy has the following levels, from most to least
+specific:
```

Comment:
Not sure I agree that the list goes from "most to least specific", but it's not a blocker.